src/views/ipv/page/pyi-triage-mobile-confirm.njk

```diff
@@ -0,0 +1,65 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% set pageTitleKey = 'pages.pyiTriageMobileConfirm.title' %}
+{% set googleTagManagerPageId = "pyiTriageMobileConfirm" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.pyiTriageMobileConfirm.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pyiTriageMobileConfirm.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#pyiTriageMobileConfirm" %}
+
+{% block content %}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pyiTriageMobileConfirm.header' | translate }}</h1>
+  <p class="govuk-body">{{ 'pages.pyiTriageMobileConfirm.content.paragraph1' | translate }}</p>
+  <p class="govuk-body">{{ 'pages.pyiTriageMobileConfirm.content.paragraph2' | translate }}</p>
+  <h2 class="govuk-heading-m">{{ 'pages.pyiTriageMobileConfirm.content.subHeading' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.pyiTriageMobileConfirm.content.paragraph3' | translate }}</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>{{ 'pages.pyiTriageMobileConfirm.content.bullet1' | translate }}</li>
+    <li>{{ 'pages.pyiTriageMobileConfirm.content.bullet2' | translate }}</li>
+  </ul>
+  <h2 class="govuk-heading-m">{{ 'pages.pyiTriageMobileConfirm.content.subHeading2' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.pyiTriageMobileConfirm.content.paragraph4' | translate }}</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>{{ 'pages.pyiTriageMobileConfirm.content.bullet3' | translate }}</li>
+    <li>{{ 'pages.pyiTriageMobileConfirm.content.bullet4' | translate }}</li>
+  </ul>
+  <h2 class="govuk-heading-m">{{ 'pages.pyiTriageMobileConfirm.content.subHeading3' | translate }}</h2>
+  <p class="govuk-body">{{ 'pages.pyiTriageMobileConfirm.content.paragraph5' | translate }}</p>
+  <p class="govuk-body">{{ 'pages.pyiTriageMobileConfirm.content.paragraph6' | translate }}</p>
+
+  <form id="pyiTriageMobileConfirmForm" action="/ipv/page/{{ pageId }}" method="POST" onsubmit="return pyiTriageMobileConfirmFormSubmit()">
```

Comment:
I don't think that onsubmit function exists? Is it needed?

Reply:
Yes you are right, I used the base of another old template which has this stuff lingering